### PR TITLE
[new] [author] Add dio

### PR DESCRIPTION
**Pull request by thysultan**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/thysultan/dio.js
 * Official website (optional, not the repository): https://thysultan.com/dio
 * NPM package url (optional): https://www.npmjs.com/package/dio.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 250
   - Count of watchers: 15
   - Count of forks: 11
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by thysultan** [comment]
Using 2-space indent and tried running package.json through fixFormat.js but the tests still fails with a format check "please use 2-spaces as indent". Any pointers?

**Comment by PeterDaveHello** [comment]
@thysultan Did you run tools/fixFormat.js under the root of cdnjs repo? Thanks.

**Comment by thysultan** [comment]
@PeterDaveHello Thanks, it's fixed now.

File: ajax/libs/dio/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "dio",
+  "filename": "dio.min.js",
+  "description": "Dio is a lightweight (~7kb) feature rich Virtual DOM framework",
+  "version": "1.0.0",
+  "homepage": "https://github.com/thysultan/dio.js",
```
Comment (PeterDaveHello) [inline ajax/libs/dio/package.json]: Please remove it if it's just the git repo, thanks.

File: ajax/libs/dio/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "type": "git",
+    "url": "https://github.com/thysultan/dio.js"
+  },
+  "npmName": "dio.js",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "dio.min.js",
+        "dio.js"
```
Comment (PeterDaveHello) [inline ajax/libs/dio/package.json]: What about `"dio*"` ?

File: ajax/libs/dio/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "url": "https://github.com/thysultan/dio.js"
+  },
+  "npmName": "dio.js",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "dio.min.js",
+        "dio.js",
+        "dio*"
```
Comment (pvnr0082t) [inline ajax/libs/dio/package.json]: We can just keep `"dio*"` and remove others. Thanks.

File: ajax/libs/dio/package.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    "browser"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/thysultan/dio.js"
+  },
+  "npmName": "dio.js",
+  "npmFileMap": [
+    {
+      "basePath": "/",
```
Comment (pvnr0082t) [inline ajax/libs/dio/package.json]: Let's use `"basePath": "",` here

File: ajax/libs/dio/package.json [hunk 5]
```diff
@@ -31,7 +31,8 @@
       "basePath": "/",
       "files": [
         "dio.min.js",
-        "dio.js"
+        "dio.js",
+        "dio*"
```
Comment (PeterDaveHello) [inline ajax/libs/dio/package.json]: Sorry, I mean to use `"dio*"` instead of `"dio.js"` and `"dio.min.js"`

Comment (thysultan) [inline ajax/libs/dio/package.json]: Yes the latest [commit](https://github.com/cdnjs/cdnjs/pull/8770/commits/dc451ddf8c7661bcefd46fd310708002d453325c) implements that change.

**Pull request merged**